vega_sim/parameter_test/parameter/configs_parameter_test.py

```diff
@@ -0,0 +1,270 @@
+from vega_sim.parameter_test.parameter.experiment import SingleParameterExperiment
+from vega_sim.parameter_test.parameter.loggers import (
+    ideal_market_maker_single_data_extraction,
+    target_stake_additional_data,
+    v1_ideal_mm_additional_data,
+    tau_scaling_additional_data,
+    limit_order_book,
+)
+from vega_sim.scenario.curve_market_maker.scenario import CurveMarketMaker
+from vega_sim.scenario.registry import IdealMarketMaker, IdealMarketMakerV2
+from vega_sim.parameter_test.parameter.loggers import (
+    BASE_IDEAL_MM_CSV_HEADERS,
+    LOB_CSV_HEADERS,
+)
+from vega_sim.parameter_test.parameter.experiment import (
+    FILE_PATTERN,
+    FILE_PATTERN_LOB,
+)
+from vega_sim.scenario.common.utils.price_process import (
+    Granularity,
+    get_historic_price_series,
+)
+
```

Comment:
What is this file for?

Reply:
I added it earlier, and then decided to not using it:) I will delete it.